spec/support/chrome_headless.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "find_chrome"
+
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: {
+      args: %w[headless disable-gpu],
```

Comment:
seems like this is the only difference btwn this file and `chrome` -- probs not worth the mental overhead, but thoughts on sharing logic?